flatcar-linux-config.yml.example

```diff
@@ -0,0 +1,5 @@
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC0g+ZTxC7weoIJLUafOgrm+h..."
```

Comment:
Can / should this be the default Vagrant insecure ssh key instead of a stub? https://www.vagrantup.com/docs/boxes/base.html#default-user-settings

Reply:
The Readme has been updated to let the user know that they need to make a copy of this file and replace their own key with it.

Reply:
But with Vagrant, users don't usually use their own ssh keys but let Vagrant manage them. The Vagrant box produces by this script should be usable by everyone download it. So I think this either shouldn't be needed at all (because packer by default does the right thing when building a vagrant box) or use a vagrant default.

Reply:
Am I missing something?

Reply:
I get what you mean now.